cmd/cue/cmd/get_go.go

```diff
@@ -788,8 +788,16 @@ func (e *extractor) reportDecl(x *ast.GenDecl) (a []cueast.Decl) {
 					}
 				}
 
+				typ := e.pkg.TypesInfo.TypeOf(name)
 				c := e.pkg.TypesInfo.Defs[v.Names[i]].(*types.Const)
 				sv := c.Val().ExactString()
+				switch t := typ.(type) {
+				case *types.Named:
+					if t.Obj().Pkg().Path() == "time" && t.Obj().Name() == "Duration" {
+						// time.Duration is an underlying int64, but CUE only accepts string.
```

Comment:
to be a bit more specific:

> // time.Duration is an int64 representing nanoseconds,
> // but CUE only accepts strings with units as accepted by time.ParseDuration.

Reply:
done in https://github.com/cue-lang/cue/pull/2984/commits/fa8b5594062955ab4b7a28676176cd55dc273e92